uber/templates/regform.html

```diff
@@ -611,4 +623,20 @@
             }
         });
     {% endif %}
+
+    {% if is_prereg_confirm_email_enabled %}
+      $(function () {
+        $('form[action="post_form"]').validate({
+          rules: {
+            email: 'required',
+            confirm_email: { equalTo: '#email' }
+          },
+          messages: {
+            confirm_email: {
+              equalTo: 'Please enter the same email address again.'
```

Comment:
Wording suggestion: "Please make sure your email address matches in the 'email' and 'confirm email' fields." This message implies they didn't fill out the confirm email field, but I expect they'll usually see this when they typo something.